src/Platform/PlatformUnix.cpp

```diff
@@ -16,6 +16,7 @@
  * You should have received a copy of the GNU General Public License
  * along with Falltergeist.  If not, see <http://www.gnu.org/licenses/>.
  */
+#if defined(__unix__) || defined(__APPLE__) || defined(__linux__)
```

Comment:
In the other project I contribute to, I was told for OS X you need `(defined (__APPLE__) && defined(__MACH__))`

I also believe `PlatformPosix` would be a better name for the file.